GMG/Scenes/Export/ExportView.swift

```diff
@@ -5,28 +5,42 @@ import UIKit
 internal import UniformTypeIdentifiers
 
 struct ExportView: View {
-    @State private var isSharing = false
-    @State private var shareItems: [Any] = []
+    @State private var model: any ExportModelStateProtocol
+    @State private var intent: any ExportIntentProtocol
+
+    init(score: Score) {
+        let model = ExportModel(score: score)
+        self._model = State(initialValue: model)
+        self._intent = State(initialValue: ExportIntent(model: model))
+    }
```

Comment:
다른 view에서 초기화 하는 방식과 통일하는게 좋을 것 같습니다~!

```suggestion
    init(score: Score) {
        let model: ExportModel = ExportModel(score: score)
        self.model = model
        self.intent = ExportIntent(model: model)
    }
```